modules/ob-ssh-authentication.adoc

```diff
@@ -0,0 +1,27 @@
+// This module is included in the following assembly:
+//
+// builds/authenticating-build-images.adoc
+
+:_content-type: CONCEPT
+[id="ob-ssh-authentication_{context}"]
+= SSH authentication
+
+With SSH authentication, you must configure the Tekton annotations to specify the hostname of the Git repository provider for use. For example, `github.com` for GitHub or `gitlab.com` for GitLab.
+
+The following example shows the usage of SSH authentication for Git:
+
+[source,yaml]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  name: secret-git-ssh-auth
+  annotations:
+    build.shipwright.io/referenced.secret: "true"
+type: kubernetes.io/ssh-auth <1>
+data:
+  ssh-privatekey: <base64 <~/.ssh/id_rsa>  <2>
```

Comment:
This is confusing with how there's a `<` in between another set of `<` and `>`. Per the `<2>` callout below, do users know what sample command is being referred to at this point? 

Looking at other `ssh-privatekey` descriptions around the repo, I see alternative styles like:

```
  ssh-privatekey: |
    # Insert ssh private key, base64 encoded
```
or
```
  ssh-privatekey: <1>
----
<1> The content of the SSH private key file.
```
Maybe something closer to one of those might work, combined/reworked with the callout you currently have?

Reply:
@apoorvajagtap Could you please look into this comment and suggest what alternative is correct to use?

Reply:
The former one looks good to proceed with.
```
  ssh-privatekey: |
    # Insert ssh private key, base64 encoded
```

Reply:
The callout no 2 can be modified to something like:
```
The `data.ssh-privatekey` is base64 encoding of the key used to authenticate into Git. This can be generated using `base64 ~/.ssh/id_rsa.pub`.
```